src/coord/src/coord.rs

```diff
@@ -1200,6 +1200,25 @@ impl Coordinator {
             .filter(|(_, frontier)| frontier != &Antichain::new())
             .collect();
 
+        for (id, frontier) in since_updates.iter() {
```

Comment:
Should this go into the not empty check block below? No idea if it'd result in meaningful benefit.